src/mumble/CMakeLists.txt

```diff
@@ -53,7 +53,14 @@ elseif(UNIX)
 		option(pipewire "Build support for PipeWire." ON)
 		option(pulseaudio "Build support for PulseAudio." ON)
 		option(speechd "Build support for Speech Dispatcher." ON)
-		option(xinput2 "Build support for XI2" ON)
+	endif()
+
+	# scripts/generate_cmake_options_docs.py does not cope with duplicate option() lines,
+	# so single out common options into combined conditions.
+	# https://github.com/mumble-voip/mumble/issues/5488
```

Comment:
That's not actually the reason why I think singling that out is better. It's just way easier to maintain if options are defined at only a single place. Otherwise you'll end up making changes to at one place, forgetting that it is also defined at a different one :)

Reply:
True -- the comment simply ought to remind future hackers in this file that the alternative way will trigger a bug, so hopefully it saves time and efforts next time.

Do you want me to change or remove the comment?

Reply:
Alright, sounds good to me :+1: 

>  Do you want me to change or remove the comment?

Nah, it's fine